src/components/mapgl/mapgl.component.ts

```diff
@@ -1172,31 +1138,36 @@ export class MapglComponent implements OnInit, AfterViewInit, OnChanges {
     });
   }
 
+  private addVisuLayers() {
+    this.visualisationsSets.visualisations.forEach((ls, v) => {
+      ls.forEach(l => this.addLayer(l));
+    });
+  }
   /**
    * @description Add layers of the selected style of each style group
    */
-  private addStylesLayers() {
-    this.mapLayers.styleGroups.forEach(styleGroup => {
-      let style;
-      const localStorageSelectedStyleId = localStorage.getItem(this.LOCAL_STORAGE_STYLE_GROUP + styleGroup.id);
-      if (localStorageSelectedStyleId) {
-        styleGroup.selectedStyle = this.getStyle(styleGroup.id, localStorageSelectedStyleId, this.mapLayers.styleGroups);
-      }
-      if (!styleGroup.selectedStyle) {
-        style = getDefaultStyle(styleGroup.styles);
-        styleGroup.selectedStyle = style;
-        localStorage.setItem(this.LOCAL_STORAGE_STYLE_GROUP + styleGroup.id, style.id);
-      } else {
-        style = styleGroup.selectedStyle;
-      }
-      if (style.geomStrategy !== undefined) {
-        this.switchLayer.next(style);
-      }
-      style.layerIds.forEach(layerId => {
-        this.addLayer(layerId);
-      });
-    });
-  }
+  // private addStylesLayers() {
```

Comment:
Do you need to keep the comments?

Reply:
For now it's kept, because I reuse some of the code blocks. I will remove it once it is completely replaced.